charts/dex.yaml

```
@@ -89,6 +90,11 @@ config:
     redirectURIs:
     - '__escape__(https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana/login/generic_oauth)'
     secret: "4lqK98NcsWG5qBRHJUqYM1"
+  - id: external-oidc-ui
+    name: '__var__(dex.spec.externalIDP.staticClient.name)'
+    redirectURIs:
+    - '__var__(dex.spec.externalIDP.staticClient.redirectURIs)'
```

To me `redirectURIs` is a list of URI so it's should be in a list here 
```suggestion
    redirectURIs: '__var__(dex.spec.externalIDP.staticClient.redirectURIs)'
```

---

Also here since it's a list we want/need to put a `| tojson` filter to this one